cli/src/cli.mjs

```diff
@@ -312,7 +312,46 @@ if ( parsed.help ) {
   }
 
 } else if ( parsed.sequence ) {
-  // @TODO
+  const eventFile = parsed.sequence;
+
+  try {
+    const sEvent = fs.readFileSync(path.resolve(__dirname, eventFile), {encoding:'utf8', flag:'r'});
+    let seqevent;
+    if ( eventFile.endsWith('yaml') || eventFile.endsWith('yml') ) {
+      seqevent = yaml.load(sEvent);
+    } else {
+      seqevent = JSON.parse(sEvent);
+    }
+    msg(`Sending sequence of events based on file ${eventFile}...`);
+    //iterating through sequence of events
+    for(let i=0;i<seqevent.length;i++) {
```

Comment:
The way things are now, you can only use the CLI to submit a sequence of events... You loop here and do <n> posts to the /sequence endpoint. But this means that someone can't create a single post request with a body like what's inside a sequence file and I think we always want to support the rest api first and then treat the CLI as "syntactic sugar" (as much as possible). So... can you instead (a) here, do a single post of the file body as-is, (b) move this loop logic down into code that runs regardless of whether the user uses a rest post call or the cli (your sequenceManagement.mjs file, I think) and obviously instead of looping and posting, loop and call sendEvent.